### ISSUE TITLE
java.lang.NoSuchMethodError: net.minecraft.block.Block.getStateForPlacement

### ISSUE THREAD

**Issue by yetshadow** [copygirl/WearableBackpacks#11]
```
[12:32:32] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: net.minecraft.block.Block.getStateForPlacement(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/util/EnumFacing;FFFILnet/minecraft/entity/EntityLivingBase;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/block/state/IBlockState;
	at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_121]
	at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_121]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:240) [bzl.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_121]
Caused by: java.lang.NoSuchMethodError: net.minecraft.block.Block.getStateForPlacement(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/util/EnumFacing;FFFILnet/minecraft/entity/EntityLivingBase;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/block/state/IBlockState;
	at net.mcft.copy.backpacks.api.BackpackHelper.placeBackpack(BackpackHelper.java:140) ~[BackpackHelper.class:?]
	at net.mcft.copy.backpacks.item.ItemBackpack.func_180614_a(ItemBackpack.java:105) ~[ItemBackpack.class:?]
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:755) ~[ForgeHooks.class:?]
	at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:144) ~[adz.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:483) ~[lv.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:679) ~[me.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:55) ~[jj.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:11) ~[jj.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[fl$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_121]
	at net.minecraft.util.Util.func_181617_a(SourceFile:45) ~[h.class:?]
	... 5 more
```

**Comment by copygirl** [comment]
Is this the 2.2.0 version or so from Curse that I removed due to an incorrect build? Can you redownload the mod from the [GitHub releases page](https://github.com/copygirl/WearableBackpacks/releases) and verify this still occurs?

**Comment by yetshadow** [comment]
1.5.0 for 1.10.2 :)

**Comment by yetshadow** [comment]
Keep crashing when I place and remove the backpack from the ground :(

Time: 2/13/17 2:16 PM
Description: Unexpected error

java.lang.NoSuchMethodError: net.minecraft.block.Block.getStateForPlacement(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/util/EnumFacing;FFFILnet/minecraft/entity/EntityLivingBase;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/block/state/IBlockState;
	at net.mcft.copy.backpacks.api.BackpackHelper.placeBackpack(BackpackHelper.java:140)
	at net.mcft.copy.backpacks.item.ItemBackpack.func_180614_a(ItemBackpack.java:105)
	at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:145)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:454)
	at PlayerControllerOF.func_187099_a(PlayerControllerOF.java:88)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1518)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2184)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1961)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1749)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1055)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.mcft.copy.backpacks.api.BackpackHelper.placeBackpack(BackpackHelper.java:140)
	at net.mcft.copy.backpacks.item.ItemBackpack.func_180614_a(ItemBackpack.java:105)
	at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:145)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:454)
	at PlayerControllerOF.func_187099_a(PlayerControllerOF.java:88)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1518)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2184)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1961)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['YETSHADOW'/1068, l='MpServer', x=154.10, y=73.00, z=253.92]]
	Chunk stats: MultiplayerChunkCache: 544, 544
	Level seed: 0
	Level generator: ID 06 - BIOMESOP, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (164,64,256), Chunk: (at 4,4,0 in 10,16; contains blocks 160,0,256 to 175,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 348 game time, 348 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 110 total; [MoCEntityFox['Fox'/51, l='MpServer', x=49.50, y=66.00, z=289.50], MoCEntityBlackBear['BlackBear'/52, l='MpServer', x=49.50, y=68.00, z=311.50], MoCEntityGrizzlyBear['GrizzlyBear'/68, l='MpServer', x=63.50, y=65.00, z=255.50], MoCEntityHorse['WildHorse'/73, l='MpServer', x=99.50, y=80.00, z=196.50], MoCEntityHorse['WildHorse'/74, l='MpServer', x=99.50, y=79.00, z=194.50], MoCEntityHorse['WildHorse'/75, l='MpServer', x=99.19, y=75.00, z=191.17], MoCEntityHorse['WildHorse'/76, l='MpServer', x=99.50, y=77.00, z=189.50], MoCEntityFox['Fox'/85, l='MpServer', x=114.50, y=77.00, z=145.50], EntityItem['item.tile.mushroom.glowshroom'/86, l='MpServer', x=111.13, y=29.00, z=301.13], EntityItem['item.tile.mushroom.glowshroom'/88, l='MpServer', x=110.13, y=29.00, z=302.13], MoCEntityFox['Fox'/90, l='MpServer', x=130.34, y=64.00, z=296.27], MoCEntityHorse['WildHorse'/95, l='MpServer', x=122.07, y=63.50, z=172.30], MoCEntityHorse['WildHorse'/96, l='MpServer', x=125.50, y=63.00, z=170.50], MoCEntityHorse['WildHorse'/97, l='MpServer', x=123.50, y=63.00, z=169.50], EntityItem['item.tile.mushroom.glowshroom'/98, l='MpServer', x=166.88, y=11.00, z=268.84], EntityWolf['Wolf'/99, l='MpServer', x=171.43, y=65.00, z=286.48], EntityWolf['Wolf'/100, l='MpServer', x=166.43, y=66.00, z=287.48], EntityWolf['Wolf'/101, l='MpServer', x=160.50, y=64.00, z=292.50], EntityWolf['Wolf'/102, l='MpServer', x=164.50, y=63.00, z=293.50], MoCEntityHorse['WildHorse'/104, l='MpServer', x=183.50, y=71.00, z=343.70], MoCEntityHorse['WildHorse'/105, l='MpServer', x=183.50, y=71.50, z=347.30], MoCEntityHorse['WildHorse'/106, l='MpServer', x=183.50, y=71.00, z=345.70], MoCEntityHorse['WildHorse'/108, l='MpServer', x=192.50, y=72.00, z=175.86], MoCEntityHorse['WildHorse'/109, l='MpServer', x=189.02, y=71.50, z=175.83], MoCEntityHorse['WildHorse'/120, l='MpServer', x=227.50, y=77.00, z=204.50], MoCEntityHorse['WildHorse'/121, l='MpServer', x=227.01, y=76.00, z=207.99], MoCEntityHorse['WildHorse'/122, l='MpServer', x=224.91, y=76.00, z=210.09], MoCEntityHorse['WildHorse'/123, l='MpServer', x=225.50, y=75.00, z=212.50], MoCEntityHorse['WildHorse'/130, l='MpServer', x=232.50, y=64.00, z=146.50], EntityWolf['Wolf'/134, l='MpServer', x=233.50, y=73.00, z=235.50], MoCEntityFox['Fox'/135, l='MpServer', x=254.50, y=70.00, z=240.50], MoCEntityGrizzlyBear['GrizzlyBear'/136, l='MpServer', x=247.50, y=68.00, z=272.50], MoCEntityGrizzlyBear['GrizzlyBear'/144, l='MpServer', x=247.89, y=68.00, z=270.99], MoCEntityHorse['WildHorse'/179, l='MpServer', x=277.08, y=89.00, z=320.81], MoCEntityHorse['WildHorse'/180, l='MpServer', x=277.50, y=82.00, z=320.50], MoCEntityHorse['WildHorse'/181, l='MpServer', x=279.19, y=89.00, z=320.75], EntityFallingBlock['Falling Block'/1089, l='MpServer', x=222.50, y=30.70, z=94.50], EntityFallingBlock['Falling Block'/1240, l='MpServer', x=189.50, y=20.89, z=140.50], EntityItem['item.tile.gravel'/5384, l='MpServer', x=91.86, y=19.00, z=193.88], EntityItem['item.tile.gravel'/5385, l='MpServer', x=122.81, y=31.00, z=306.13], EntityFallingBlock['Falling Block'/1300, l='MpServer', x=184.50, y=19.89, z=142.50], EntityFallingBlock['Falling Block'/1304, l='MpServer', x=187.50, y=19.89, z=139.50], EntityFallingBlock['Falling Block'/1307, l='MpServer', x=192.50, y=19.89, z=142.50], EntityItem['item.tile.gravel'/5410, l='MpServer', x=165.45, y=17.00, z=264.45], EntityItem['item.tile.gravel'/5464, l='MpServer', x=91.75, y=19.00, z=192.77], EntityItem['item.tile.gravel'/5540, l='MpServer', x=141.45, y=38.00, z=234.96], EntityItem['item.tile.gravel'/5541, l='MpServer', x=180.88, y=21.00, z=222.55], EntityFallingBlock['Falling Block'/1450, l='MpServer', x=153.50, y=9.70, z=94.50], EntityCreeper['Creeper'/5553, l='MpServer', x=164.50, y=35.00, z=203.50], EntityCreeper['Creeper'/5554, l='MpServer', x=163.50, y=35.00, z=203.50], EntityItem['item.tile.gravel'/5580, l='MpServer', x=141.29, y=38.00, z=232.88], EntitySkeleton['Skeleton'/5677, l='MpServer', x=183.50, y=26.00, z=227.50], EntitySkeleton['Skeleton'/5678, l='MpServer', x=180.50, y=26.00, z=227.50], EntityCreeper['Creeper'/5753, l='MpServer', x=109.50, y=34.00, z=247.50], MoCEntityMiniGolem['MiniGolem'/5776, l='MpServer', x=228.50, y=29.00, z=280.50], MoCEntityMiniGolem['MiniGolem'/5777, l='MpServer', x=225.50, y=29.00, z=281.50], EntityItem['item.tile.flower_1.miners_delight'/5908, l='MpServer', x=114.64, y=12.00, z=200.80], EntityCreeper['Creeper'/5981, l='MpServer', x=231.50, y=24.00, z=267.50], EntityCreeper['Creeper'/5982, l='MpServer', x=231.50, y=24.00, z=267.50], EntityItem['item.item.seeds'/5995, l='MpServer', x=212.01, y=68.50, z=221.58], EntitySpider['Spider'/6054, l='MpServer', x=84.50, y=61.00, z=326.50], EntityZombie['Zombie'/6055, l='MpServer', x=83.50, y=61.00, z=328.50], EntityZombie['Zombie'/6056, l='MpServer', x=84.50, y=61.00, z=326.50], MoCEntitySilverSkeleton['SilverSkeleton'/6070, l='MpServer', x=112.50, y=35.00, z=247.50], EntityZombie['Zombie'/6073, l='MpServer', x=97.50, y=37.00, z=262.50], EntityZombie['Zombie'/6074, l='MpServer', x=96.50, y=37.00, z=263.50], EntityZombie['Zombie'/6075, l='MpServer', x=96.51, y=37.00, z=260.60], EntityZombie['Zombie'/6076, l='MpServer', x=96.50, y=37.00, z=264.50], EntityItem['item.tile.flower_0.blue_hydrangea'/6127, l='MpServer', x=159.22, y=79.00, z=240.63], EntitySkeleton['Skeleton'/6136, l='MpServer', x=159.50, y=37.00, z=198.50], EntityZombie['Zombie'/6137, l='MpServer', x=161.50, y=37.00, z=197.50], MoCEntityWerewolf['Werewolf'/6158, l='MpServer', x=110.50, y=29.00, z=304.50], EntitySkeleton['Skeleton'/6191, l='MpServer', x=188.50, y=34.00, z=180.50], EntityItem['item.item.carrots'/6239, l='MpServer', x=216.10, y=66.00, z=246.84], EntityCreeper['Creeper'/6275, l='MpServer', x=149.50, y=49.00, z=275.50], EntitySpider['Spider'/6290, l='MpServer', x=93.50, y=24.00, z=331.50], EntitySpider['Spider'/6291, l='MpServer', x=94.50, y=24.00, z=333.50], EntityZombie['Zombie'/6293, l='MpServer', x=92.50, y=24.00, z=329.50], EntityZombie['Zombie'/6309, l='MpServer', x=115.25, y=41.00, z=230.51], EntityPlayerSP['YETSHADOW'/1068, l='MpServer', x=154.10, y=73.00, z=253.92], EntitySkeleton['Skeleton'/6396, l='MpServer', x=113.50, y=40.00, z=235.50], EntityCreeper['Creeper'/6498, l='MpServer', x=119.50, y=37.00, z=320.50], EntityItem['item.tile.plant_1.cattail'/6551, l='MpServer', x=202.86, y=62.00, z=299.14], EntityZombie['Zombie'/6581, l='MpServer', x=84.50, y=61.00, z=326.50], MoCEntityFly['Fly'/2569, l='MpServer', x=35.50, y=70.00, z=258.50], EntityZombie['Zombie'/6762, l='MpServer', x=136.50, y=39.00, z=226.50], EntityZombie['Zombie'/6787, l='MpServer', x=100.50, y=27.00, z=331.50], EntityItem['item.tile.plant_1.cattail'/6792, l='MpServer', x=169.26, y=63.00, z=289.13], EntitySpider['Spider'/6829, l='MpServer', x=220.50, y=28.00, z=242.50], EntitySkeleton['Skeleton'/6835, l='MpServer', x=109.50, y=36.00, z=253.50], EntityZombie['Zombie'/6897, l='MpServer', x=143.50, y=30.00, z=216.50], EntityItem['item.tile.flower_1.miners_delight'/6926, l='MpServer', x=153.00, y=17.00, z=244.91], EntityItem['item.tile.flower_1.miners_delight'/6927, l='MpServer', x=151.66, y=17.00, z=244.13], EntityItem['item.tile.plant_1.cattail'/6944, l='MpServer', x=156.88, y=63.00, z=294.00], MoCEntitySnail['Snail'/3352, l='MpServer', x=110.50, y=70.00, z=262.50], MoCEntitySnail['Snail'/3353, l='MpServer', x=110.50, y=70.00, z=267.50], MoCEntitySnail['Snail'/3354, l='MpServer', x=106.50, y=70.00, z=266.50], MoCEntitySnail['Snail'/3355, l='MpServer', x=108.50, y=70.00, z=261.50], MoCEntityCricket['Cricket'/3780, l='MpServer', x=170.50, y=64.00, z=149.50], MoCEntityCricket['Cricket'/3782, l='MpServer', x=171.50, y=64.00, z=149.50], MoCEntityCricket['Cricket'/3783, l='MpServer', x=172.50, y=64.00, z=146.50], MoCEntityCricket['Cricket'/3784, l='MpServer', x=172.50, y=64.00, z=146.50], MoCEntityButterfly['ButterFly'/3786, l='MpServer', x=194.50, y=67.00, z=160.50], MoCEntityCricket['Cricket'/3789, l='MpServer', x=165.50, y=64.00, z=322.50], MoCEntityCricket['Cricket'/3790, l='MpServer', x=166.50, y=64.00, z=321.50], MoCEntityMaggot['Maggot'/3791, l='MpServer', x=250.50, y=71.00, z=191.50], MoCEntityButterfly['ButterFly'/3793, l='MpServer', x=249.50, y=71.00, z=187.50], MoCEntityFly['Fly'/3795, l='MpServer', x=126.50, y=79.00, z=237.50], MoCEntityAnt['Ant'/3797, l='MpServer', x=42.50, y=66.00, z=129.50], MoCEntityAnt['Ant'/3798, l='MpServer', x=44.50, y=66.00, z=129.50]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:524)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2660)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:400)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_121, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1895015720 bytes (1807 MB) / 3568189440 bytes (3402 MB) up to 4281597952 bytes (4083 MB)
	JVM Flags: 7 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128m -Xmx1024m -Xmx4G
	IntCache: cache: 0, tcache: 0, allocated: 4, tallocated: 105
	FML: MCP 9.32 Powered by Forge 12.18.2.2099 Optifine OptiFine_1.10.2_HD_U_D6 33 mods loaded, 33 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.2.2099.jar) 
	UCHIJAAAA	Forge{12.18.2.2099} [Minecraft Forge] (forge-1.10.2-12.18.2.2099.jar) 
	UCHIJAAAA	buildingbrickshooks{1.10.2-2.0.13} [Building Bricks Hooks] (minecraft.jar) 
	UCHIJAAAA	Quark{r1.1-70} [Quark] (Quark-r1.1-70.jar) 
	UCHIJAAAA	AutoRegLib{1.0-2} [AutoRegLib] (AutoRegLib-1.0-2.jar) 
	UCHIJAAAA	BiblioCraft{2.2.1} [BiblioCraft] (BiblioCraft[v2.2.1][MC1.10.2].jar) 
	UCHIJAAAA	BiomesOPlenty{5.0.0.2142} [Biomes O' Plenty] (BiomesOPlenty-1.10.2-5.0.0.2142-universal.jar) 
	UCHIJAAAA	bookshelf{1.4.3.338} [Bookshelf] (Bookshelf-1.10.2-1.4.3.338.jar) 
	UCHIJAAAA	buildingbricks{1.10.2-2.0.13} [Building Bricks] (BuildingBricks-1.10.2-2.0.13.jar) 
	UCHIJAAAA	cfm{4.1.2} [MrCrayfish's Furniture Mod] (cfm-4.1.2-mc1.10.2.jar) 
	UCHIJAAAA	chisel{MC1.10.2-0.0.7.7} [Chisel] (Chisel-MC1.10.2-0.0.7.7.jar) 
	UCHIJAAAA	chiselsandbits{12.11} [Chisels & Bits] (chiselsandbits-12.11.jar) 
	UCHIJAAAA	crafttweakerjei{1.0.1} [CraftTweaker JEI Support] (CraftTweaker-1.10.2-3.0.20.jar) 
	UCHIJAAAA	MineTweaker3{3.0.20} [MineTweaker 3] (CraftTweaker-1.10.2-3.0.20.jar) 
	UCHIJAAAA	ctgui{1.0.0} [CT-GUI] (CraftTweaker-1.10.2-3.0.20.jar) 
	UCHIJAAAA	mocreatures{10.0.6} [DrZhark's Mo'Creatures Mod] (DrZharks MoCreatures Mod-10.0.6.jar) 
	UCHIJAAAA	eplus{4.1.0.113} [Enchanting Plus] (EnchantingPlus-1.10.2-4.1.0.115.jar) 
	UCHIJAAAA	guideBook{1.6.1} [Guide Book] (GuideBook-1.6.3.jar) 
	UCHIJAAAA	levelup{${version}} [Level Up!] (LevelUp!-0.12.8-1.10.2.jar) 
	UCHIJAAAA	mantle{1.10.2-1.1.4.203} [Mantle] (Mantle-1.10.2-1.1.4.jar) 
	UCHIJAAAA	MapWriter{2.6.19} [MapWriter] (MapWriter-1.10.2-2.6.19.jar) 
	UCHIJAAAA	missing_pieces{4.1.3} [Missing Pieces] (missing_pieces-1.10.2-4.1.3.jar) 
	UCHIJAAAA	Neat{1.3-12} [Neat] (Neat 1.3-12.jar) 
	UCHIJAAAA	offlineskins{1.10.2-v2a} [OfflineSkins] (OfflineSkins-1.10.2-v2a.jar) 
	UCHIJAAAA	railcraft{10.0.1} [Railcraft] (Railcraft_1.10.2-10.0.1.jar) 
	UCHIJAAAA	redstonepaste{1.7.4} [Redstone Paste] (redstonepaste-mc1.9.4-1.7.4.jar) 
	UCHIJAAAA	Roguelike{1.5.6} [Roguelike Dungeons] (RoguelikeDungeons-1.10.2-1.5.6.jar) 
	UCHIJAAAA	tconstruct{1.10.2-2.5.6b.jenkins424} [Tinkers' Construct] (TConstruct-1.10.2-2.5.6b.jar) 
	UCHIJAAAA	waystones{2.0.10} [Waystones] (Waystones_1.10.2-2.0.10.jar) 
	UCHIJAAAA	wearablebackpacks{1.5.0} [Wearable Backpacks] (WearableBackpacks-1.10.2-1.5.0 (1).jar) 
	UCHIJAAAA	examplemod{1.0} [examplemod] (WTF-Expedition_1.10.2_v-1.4.1.jar) 
	UCHIJAAAA	wtfcore{1.10.2_v1.4} [WTFs-Expedition] (WTF-Expedition_1.10.2_v-1.4.1.jar) 
	Loaded coremods (and transformers): 
LoadingPlugin (Quark-r1.1-70.jar)
  vazkii.quark.base.asm.ClassTransformer
NWRTweak (redstonepaste-mc1.9.4-1.7.4.jar)
  net.fybertech.nwr.NWRTransformer
offlineskins (OfflineSkins-1.10.2-v2a.jar)
  lain.mods.skins.asm.ASMTransformer
BookshelfLoadingPlugin (Bookshelf-1.10.2-1.4.3.338.jar)
  net.darkhax.bookshelf.asm.BookshelfTransformerManager
ChiselCorePlugin (Chisel-MC1.10.2-0.0.7.7.jar)
  team.chisel.common.asm.ChiselTransformer
LoadingPluginBuildingBricks (BuildingBricks-1.10.2-2.0.13.jar)
  com.hea3ven.buildingbricks.core.load.ClassTransformerBuildingBricks
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 378.49' Renderer: 'GeForce GTX 1070/PCIe/SSE2'
	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)
		- chiselsandbitsIntegration (Enabled/Not Forced)

	Launched Version: 1.10.2
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 1070/PCIe/SSE2 GL version 4.5.0 NVIDIA 378.49, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: §9L§fR§cQ§f.zip (incompatible)
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
	OptiFine Version: OptiFine_1.10.2_HD_U_D6
	Render Distance Chunks: 12
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: KUDA-Shaders v6.4.80.zip
	OpenGlVersion: 4.5.0 NVIDIA 378.49
	OpenGlRenderer: GeForce GTX 1070/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8

**Comment by copygirl** [comment]
Hmm.. can you try updating to Forge 12.18.3.2185?

**Comment by yetshadow** [comment]
Sorry for the late response, but yeah, the update to 2185 of forge has fixed the bug, sorry for the late, I'm using a custom launcher and the change of forge version is hard to do, thank you for your time :)

**Comment by copygirl** [comment]
Ah, good. Going to leave this issue up as a reminder to require Forge 2185 in the next version of WB.

**Comment by copygirl** [comment]
Fixed in cbf5de6a8e452bf1c7066af696889deb31afa816.
